integration/tests/cook/test_basic.py

```diff
@@ -1561,6 +1582,10 @@ def check_unscheduled_reason():
         finally:
             util.kill_jobs(self.cook_url, [job_uuid_1, job_uuid_2])
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
     def test_unscheduled_jobs_partial(self):
         unsatisfiable_constraint = ['HOSTNAME', 'EQUALS', 'fakehost']
```

Comment:
Is it possible to use a different constraint that causes jobs to not be scheduled and avoid the issue of launching hosts that never get used?

Reply:
:+1:

Reply:
This didn't actually need the constraint, so I removed it